tests/functional/testOnboarding.js

```diff
@@ -277,4 +298,73 @@ describe('Onboarding', function() {
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.ONBOARDING_VIEW, 'currentIndex'), 3);
     assert.equal(await vpn.getSetting('onboardingStep'), 3);
   });
+
+  //Tests that the data collection checkbox is bound to onboardingDataCollectionEnabled, and it's default state
+  it('Data collection checkbox default and binding', async () => {
```

Comment:
nit: Instead of having this separate test case, I'd check the default and binding in all the tests related to the checkbox. That approach will catch possible issues with the resetting of the checkbox.